templates/terraform/unordered_list_customize_diff.erb

```diff
@@ -0,0 +1,44 @@
+keys := diff.GetChangedKeysPrefix(<%= go_literal(Google::StringUtils.underscore(prop.name)) -%>)
+if len(keys) < 1 {
+	return nil
+}
+
+oldCount, newCount := diff.GetChange("<%= Google::StringUtils.underscore(prop.name) -%>.#")
+var count int
+if oldCount.(int) < newCount.(int) {
+    count = newCount.(int)
+} else {
+    count = oldCount.(int)
+}
+if count < 1 {
+    return nil
+}
+
+if count < 1 {
```

Comment:
This is the same as the lines immediately before it

Reply:
Fixed!